marketch.sketchplugin/Contents/Sketch/util.cocoascript

```diff
@@ -261,7 +261,7 @@ var util = {
 
     //NSString to js string
     toJSString: function(NSStr){
-        return new String(encodeURI(NSStr)).toString();
+        return new String(encodeURI(trim(NSStr))).toString();
```

Comment:
just use `console.log` as in javascript: 
<img width="572" alt="2015-09-13 11 29 09" src="https://cloud.githubusercontent.com/assets/1134658/9835085/ac74a78c-5a0a-11e5-8dc1-45c288b88ae8.png">
then you can see a window with the logs:
![2015-09-13 11 30 47](https://cloud.githubusercontent.com/assets/1134658/9835091/edc4f368-5a0a-11e5-8209-1c3c6289dff0.png)